fields/field.textbox.php

```diff
@@ -75,9 +75,9 @@ public function isSortable() {
 	-------------------------------------------------------------------------*/
 
 		public function createHandle($value, $entry_id) {
-			$handle = Lang::createHandle(strip_tags(html_entity_decode($value)));
+			$handle = Lang::createHandle(strip_tags(html_entity_decode($value)),160);
```

Comment:
Ah; I don't recall why I did this to 160... must have been a change I did earlier whilst trying out utf8-mb4 or something of the sort; maybe handle length should be configurable?

Any reason it should be as long as 1024? If used within a URL (link) isn't that too long?